.circleci/config.yml

```diff
@@ -9,6 +9,10 @@ parameters:
     description: The version of react to be used
     type: string
     default: stable
+  node-version:
+    description: The version of node to be used
+    type: string
+    default: '23'
```

Comment:
What's the reason for bumping node version?

Reply:
There's CJS/ESM issues everywhere, using an up-to-date node version fixes some of them.